ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -145,6 +145,21 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-vsphere-ovn-hybrid-custom-vxlan-port
+- as: e2e-network-migration
+  steps:
+    cluster_profile: aws
+    test:
+    - ref: ovn-sdn-migration
+    - ref: openshift-e2e-test
```

Comment:
> Looks like the workflows created here are implying that e2e is done, but I didn't see that happening in the code. I think that
> would be ideal though; to run e2e tests after the migration and/or rollback happens.

@jluhrsen There is no test defined in the workflow `openshift-e2e-aws-ovn-sdn-migration` itself. Instead, I refer the `openshift-e2e-test` here. So that the workflow can be reused by both migration and rollback jobs.